README.md

```diff
@@ -1,141 +1,146 @@
 
-This approach to creating a low volatility currency is closest to the work being done by [Fragments](https://www.fragments.org/protocol/), but is also inspired by [Maker](https://makerdao.com/), [StableUnit](https://stableunit.org), and [Basis](http://www.basis.io/). As research in this area matures, we expect that there will be many low-volatility currencies available with slightly different risks, rewards, and utilities that lead to their adoption as the preferred medium of exchange and store of value for specific communities. Within the Aragon Network jurisdiction, the marginal utility and rewards of participation in governance will lead to ANT becoming the preferred reserve currency for collateralizing agreements.
-## 4.1 Price Feed Oracle
-Agents can agree to act as price feed oracles by collateralizing an agreement with ANT that commits them to provide accurate price information and not attempt to manipulate the results of the feed. We can take the median value from all participants weighted by their collateral to get a price for a specific period.
+**Redistributing Collateral**
 
-If an agent tries to manipulate the price feed, a dispute can be created and the agent's collateral taken. Since price information is widely available and fairly objective, this mechanism should provide reasonably strong guarantees under the assumption there is at least one honest agent monitoring the price feed.
+If there was any appeal rounds before the final ruling, the total amount of collateral that was deposited for triggering each round will be assigned to the appealer supporting the final ruling.
 
-Agents which provide the price feed take on risk by collateralizing an agreement and must be compensated both for that risk and for providing a service to the network. A portion of inflation is dynamically allocated to incentivizing this behavior based on a target level of participation.
-## 4.2 Stability Reserve
-The purpose of the stability reserve is to dampen the volatility of ANT, particularly on the downside, in order to ensure that it can be effectively used as a unit of account and store of value for collateralizing agreements. By optimizing for store of value and unit of account properties, it is likely that ANT could also become a common medium of exchange.
+In the first adjudication round of a dispute (the original round created with the dispute), the Court doesn't directly manage collateral (as there may not be collateral at all or a different token is used). In case that there was collateral at stake depending on the Court ruling, whenever the Court has a final ruling it will notify the contract being *arbitered* with the ruling and this contract can then distribute collateral.
```

Comment:
```suggestion
In the first adjudication round of a dispute (the original round created with the dispute), the Court doesn't directly manage collateral (as there may not be collateral at all or a different token is used). In cases where there is collateral at stake depending on the Court ruling, whenever the Court has a final ruling it will notify the contract being *arbitrated* with the ruling and this contract can then redistribute the collateral.
```